src/js/components/commits/list.tsx

```diff
@@ -39,8 +39,9 @@ const AuthorTableCell = ({ item, className, ...props }: TableCellProps) => {
     return null;
   }
   let author = item.author.username;
-  if (item.author.name && item.author.name !== author) {
-    author = `${author} (${item.author.name})`;
+  const name = item.author?.name;
```

Comment:
Ooh fancy.

Reply:
It's really all your fault... https://oddbird.slack.com/archives/C1310AQNA/p1577730958000500 😄